style.css

```diff
@@ -342,3 +349,101 @@ nav .status-bar {
   font-size: 22px;
   line-height: 43px;
 }
+
+.contact-section {
+  background: white;
+}
```

Comment:
- [ ] Great job 👍. It's better to layout this part using `flexbox`, as it is one of the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements).